containers/submit/form-section.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react'
+import { Flex, Box, Field } from 'blockstack-ui'
+import { Select } from '@components/mining/select'
+
+export const FormSection = ({ state, fields, handleChange, errors, message, setState }) => (
+  <>
+    {fields.map((field) => {
+      if (field.type === 'radio') {
+        if (!field.options || field.options.length !== 2) {
+          console.log('Radio type fields need 2 options (true/false)')
+          return null
+        }
+        return (
+          <React.Fragment key={`radio-${field.name}`}>
+            <Field.LabelAdvanced
+              pb={3}
+              label={field.label}
+              required={field.required}
+              error={errors && field && errors[field.name] && errors[field.name]}
+            />
+            <Box pb={4}>
```

Comment:
Should we add a `field.options.length` conditional to render this box with paddingBottom? if the array is empty, it will still display the padding.

Reply:
done